components/resources/brave_components_strings.grd

```diff
@@ -241,6 +241,7 @@
       <message name="IDS_BRAVE_REWARDS_LOCAL_GENERAL_GRANT_ERROR_TITLE" desc="">Oops, something is wrong. Please try again later.</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_GENERAL_GRANT_ERROR_BUTTON" desc="">OK</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_GENERAL_GRANT_ERROR_TEXT" desc="">Brave Rewards is having an issue. Please try again later.</message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_WALLET_CORRUPTED" desc="">We detected that something is wrong with your wallet. Please try to recover it.</message>
```

Comment:
Should we use the same text as BL?:

```
Unfortunately your active wallet has been corrupted. You must recover your backup wallet with your recovery keys before any transactions can be processed. We apologize for the inconvenience.
```

Reply:
let me see if we can use this much text

Reply:
Sorry. It was actually in more than one string setting:
```
corruptedOverlayMessage=Unfortunately your active wallet has been corrupted.
corruptedOverlayText=You must recover your backup wallet with your recovery keys before any transactions can be processed. We apologize for the inconvenience.
```

Reply:
![image](https://user-images.githubusercontent.com/9574457/48718210-360e9e80-ec1b-11e8-881a-0f62a2f205e9.png)

it's quite big, do you have suggestion how to shrink it?

Reply:
Should we have the close button available when we have a corrupt wallet? Also, take off 'Unfortunately'. I think it's implied that a corrupt wallet is unfortunate. Also take off 'with your recovery keys'. I would say take off the apology sentence but I think product would want to chime on that.

Reply:
Keep 'Uh oh!'

Reply:
Try this:
`Uh oh! We detected that something is wrong with your wallet. You must recover your backup wallet before any transactions can be processed`

Reply:
sure

Reply:
done